api/organizations.go

```diff
@@ -397,6 +397,45 @@ func (a *API) acceptOrganizationMemberInvitationHandler(w http.ResponseWriter, r
 	httpWriteOK(w)
 }
 
+func (a *API) pendingOrganizationMembersHandler(w http.ResponseWriter, r *http.Request) {
+	// get the user from the request context
+	user, ok := userFromContext(r.Context())
+	if !ok {
+		ErrUnauthorized.Write(w)
+		return
+	}
+	// get the organization info from the request context
+	org, _, ok := a.organizationFromRequest(r)
+	if !ok {
+		ErrNoOrganizationProvided.Write(w)
+		return
+	}
+	if !user.HasRoleFor(org.Address, db.AdminRole) {
+		ErrUnauthorized.Withf("user is not admin of organization").Write(w)
+		return
+	}
+	// check if the user is already verified
+	if !user.Verified {
```

Comment:
@lucasmenendez  why is that relevenat here?

Reply:
to prevent that not verified users to perform actions

Reply:
But what is the action in this case? I mean you have check that the user is Admin, and your a getting the pending invitations...

Reply:
The operation is to get a list of pending invitations, and yes, I've already checked that the user has an admin role for the requested org, but I think if the account is not yet verified, the user should not perform any operation, not even list invitations.

Reply:
@lucasmenendez will create new PR that will improve the authorization/verifcation  helpers of the API